docs/api/tabs.md

```diff
@@ -42,448 +26,34 @@ Both `ion-tabs` and `ion-tab-bar` can be used as standalone elements. They don
 
 The `ion-tab-bar` needs a slot defined in order to be projected to the right place in an `ion-tabs` component.
 
-## Interfaces
-
-### TabsCustomEvent
-
-While not required, this interface can be used in place of the `CustomEvent` interface for stronger typing with Ionic events emitted from this component.
+## Usage with Router
 
-```typescript
-interface TabsCustomEvent extends CustomEvent {
-  detail: { tab: string };
-  target: HTMLIonTabsElement;
-}
-```
+Tabs can be used with the Ionic router to implement tab-based navigation. The tab bar and active tab will automatically resolve based on the url. This is the most common pattern for tabs navigation.
```

Comment:
Could we mention that using either `ion-router-outlet` or `ion-nav` is required for `ion-tabs` to work?